benchmarks/benchmark.py

```diff
@@ -62,31 +62,70 @@ def main():
     techniques used in the aforementioned paper.
     """
 
-    score_seed = 1
-    num_samples = 10000
-    archive = CVTArchive(
-        solution_dim=20,
-        cells=512,
-        ranges=[(0., 1.), (0., 1.)],
-    )
-    cvt_centroids = archive.centroids
+    # default settings to be used for all benchmarking tests
+    score_seed = 1823170571
+    num_samples = 100000
+
+    # original kmeans centroid generation
+    kmeans_archive = CVTArchive(solution_dim=20,
```

Comment:
Can you make these into a for loop? You can just iterate over a list of different values for `centroid_method` rather than repeating code.